### ISSUE TITLE
sort-prop-types with callbacksLast returns the same error multiple times

### ISSUE THREAD

**Issue by trygveaa** [yannickcr/eslint-plugin-react#519]
When using the rule `react/sort-prop-types` with `callbacksLast` set to `true`, it sometimes returns the same error multiple times instead of showing different errors. For example with this code:

```
var Component = React.createClass({
  propTypes: {
    onChange: React.PropTypes.func,
    a: React.PropTypes.string,
    c: React.PropTypes.string,
    b: React.PropTypes.string,
  }
});
```

It gives this output:

```
  3:5  error  Callback prop types must be listed after all other prop types  react/sort-prop-types
  3:5  error  Callback prop types must be listed after all other prop types  react/sort-prop-types
  3:5  error  Callback prop types must be listed after all other prop types  react/sort-prop-types
```

The error for the position of `onChange` should only be printed once, and it should report that `b` or `c` is at the wrong position.
